### DISCUSSION TITLE
Save/Open Project As XML/JSON File Feature

### DISCUSSION THREAD

**Discussion by MohamedAshref371** [AngryCarrot789/FramePFX#7] [Ideas]
Save/Open Project As XML/JSON File Feature.

This feature will make it easy to automatically create save files outside the program and then open them in it.

**Comment by AngryCarrot789** [comment]
I like this idea, might have to replace the binary serialisation with JSON in order to get it working though

**Reply by MohamedAshref371** [reply]
You don't have to replace anything, just add an additional option. ❤
